weblate_web/models.py

```diff
@@ -622,7 +622,7 @@ def get_suggestions(self):
                     (
                         "extended",
                         _("Extended support"),
-                        _("Don’t be just Basic, get a worry-free package."),
```

Comment:
This isn't a bad idea, but the translations are cringeworthy.

Reply:
considered